main.tf

```diff
@@ -49,7 +49,8 @@ module "elastic_beanstalk_environment" {
   env_vars = "${
       merge(
         map(
-          "EFS_HOST", "${module.efs.dns_name}",
+          "EFS_HOST", "${var.use_efs_ip_address ? module.efs.mount_target_ips[0] : module.efs.dns_name}",
+          "USE_EFS_IP", "${var.use_efs_ip_address}",
```

Comment:
What is the use case of `USE_EFS_IP` variable in Beanstalk environment?

Reply:
We get value of this ENV variable and using IP address of EFS mount point instead DNS name if `USE_EFS_IP = 1`